sdk/eventhub/Azure.Messaging.EventHubs.Processor/tests/Diagnostics/DiagnosticsTests.cs

```diff
@@ -152,7 +155,23 @@ async IAsyncEnumerable<PartitionEvent> mockPartitionEventEnumerable()
             Assert.That(listener.Scopes.Select(s => s.Name), Has.All.EqualTo(DiagnosticProperty.EventProcessorProcessingActivityName));
             Assert.That(listener.Scopes.SelectMany(s => s.Links), Has.One.EqualTo("id"));
             Assert.That(listener.Scopes.SelectMany(s => s.Links), Has.One.EqualTo("id2"));
-            Assert.That(listener.Scopes.SelectMany(s => s.Activity.Tags), Has.Exactly(2).EqualTo(new KeyValuePair<string, string>(DiagnosticProperty.KindAttribute, DiagnosticProperty.ConsumerKind)), "The activities tag should be server.");
+
+            var expectedTags = new List<KeyValuePair<string, string>>()
```

Comment:
I think you can use CollectionAsserts.Equal here

Reply:
`Assert.That(listener.Scopes, Is.EquivilentTo(scope.Activity.Tags.ToList())` should also work.

Reply:
We can't because it seems other tags can be added behind the scenes (namely `["az.namespace", "Microsoft.EventHub"]`). I can use `CollectionAssert.IsSubsetOf`, though.

Reply:
In which case:

`Assert.That(thing, Is.SubsetOf(otherThign))`   :smile: